configure.go

```diff
@@ -0,0 +1,45 @@
+package discovery
+
+import (
+	"fmt"
+	"net/url"
+	"os"
+	"strings"
+
+	kv "gopkg.in/clever/kayvee-go.v2"
+)
+
+// m is a convenience type for using kv.
+type m map[string]interface{}
+
+const (
+	templateVar = "SERVICE_%s_%s_%%s"
+)
+
+func getVar(envVar string) string {
+	envVar = strings.ToUpper(envVar)
```

Comment:
nit: env variable names are case sensitive, I think? We don't use any lowercase env variable names so this is probably fine